modules/extensions-spring-support/src/test/resources/schemas/implicit-config.xsd

```diff
@@ -101,6 +103,7 @@
       <xs:extension xmlns:mule="http://www.mulesoft.org/schema/mule/core" base="mule:abstractOperatorType">
         <xs:sequence minOccurs="0" maxOccurs="1">
           <xs:element minOccurs="0" maxOccurs="1" ref="mule:abstract-reconnection-strategy"></xs:element>
+          <xs:element minOccurs="0" maxOccurs="unbounded" ref="mule:error-mapping"></xs:element>
```

Comment:
this is weird. Your original intent was to reflect this on the extension model, but why is the XSD being changed? Wasn't it possible to provide this error mapping before?

Reply:
previously this was in the mule xsd, on the parent element of all operations. Since the xsd generation is fed from the ext model, it is reflected here. The mule xsd was changed accordingly.

Reply:
I don't think this is a good change then. If it can be defined globally in the base type, it should be kept like that. Not only because of redundancy and code tidyness... this change risks potential problems because of element ordering in exchange for no gain I can appretiate